client/python/requirements.txt

```diff
@@ -3,4 +3,4 @@ httpx>=0.24.1
 huggingface_hub>=0.19.3
 packaging
 typing_extensions~=4.0
-websockets>=10.0,<13.0
+websockets>=10.0,<15.0
```

Comment:
I'd like to do a 2.0 `gradio_client` release soon where we drop `websockets` as a dependency and end support for 3.x altogether. But in the meantime